santa-tracker-base.conf

```diff
@@ -37,31 +37,31 @@ enableVfsRetention = [
 ]
 
 noOptimizations = ${base} {
-    title = "Baseline"
+    title = "0 – Baseline"
     tasks = [":santa-tracker:assembleDebug"]
 
     apply-non-abi-change-to = ${file-to-change}
     # apply-abi-change-to = ${file-to-change}
     # apply-android-resource-value-change-to = ${resource-to-change}
 }
 
-onlyInstantExecution = ${noOptimizations} {
-    title = "Instant execution"
-    gradle-args = ${noOptimizations.gradle-args} ${enableInstantExecution}
+onlyPartialInvalidation = ${noOptimizations} {
```

Comment:
I wonder if we should get rid of this scenario as well.

Reply:
I removed it.